eth/handler_trust.go

```diff
@@ -34,6 +33,13 @@ func (h *trustHandler) PeerInfo(id enode.ID) interface{} {
 func (h *trustHandler) Handle(peer *trust.Peer, packet trust.Packet) error {
 	switch packet := packet.(type) {
 	case *trust.RootResponsePacket:
+		verifyResult := &core.VerifyResult{
+			Status:      packet.Status,
+			BlockNumber: packet.BlockNumber,
+			BlockHash:   packet.BlockHash,
+			Root:        packet.Root,
+		}
+		h.Chain().VerifyManger().HandleRootResponse(verifyResult, peer.ID())
```

Comment:
return h.Chain().VerifyManger().HandleRootResponse(verifyResult, peer.ID()) and delete TODO comments here.

Reply:
Sorry, return nil is fine, as the connection should keep online, just delete the TODO pls.